Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/data/model/Tasks.java

```diff
@@ -22,6 +22,14 @@ public static Tasks empty() {
         return from(ImmutableMapWithCopy.<Id, SyncedData<Task>>empty());
     }
 
+    public static Tasks from(ImmutableList<SyncedData<Task>> tasks) {
+        ImmutableMap.Builder<Id, SyncedData<Task>> builder = ImmutableMap.builder();
+        for (SyncedData<Task> task : tasks) {
+            builder.put(task.data().id(), SyncedData.from(task.data(), task.syncState(), task.lastSyncAction()));
```

Comment:
Done in https://github.com/novoda/spikes/pull/113/commits/fe9a9d11ad7afecd4bfa7ad5d03e5a15dc5575b1